charts/stellar-disbursement-platform/templates/02.2-deployment-ap.yaml

```diff
@@ -42,7 +42,7 @@ spec:
         - name: {{ .Chart.Name }}-ap
           securityContext:
             {{- tpl (toYaml .Values.anchorPlatform.deployment.securityContext) . | nindent 12 }}
-          image: "stellar/anchor-platform:2.1.3"
+          image: "stellar/anchor-platform:2.6.2"
```

Comment:
:question: is this hardcoded on purpose?

Reply:
Yes we pin this version on purpose because that's the one we tested with. We have to run regression tests every time we upgrade the anchor version for an SDP.

Reply:
Hmm this actually introduces more problems. You don't allow users to use internal docker registries, you don't allow them to use custom builds, you don't allow them to run images build from source.
You also don't give users ability to apply security patches without upgrading the whole chart which could slow things down.

I think what you're trying to achieve could be better solved by having a default value set to what you want but to still allow users to use another image if they choose or have to do so

Reply:
That's a good point, I'll address that. Thank you!

Reply:
Addressed in a1f517df7347ea36969c8a94180419c615201df5